### PR TITLE
Release Candidate 3.0.2-rc0

### PR DESCRIPTION
**Pull request by AndreasFuchsSIT**
Signed-off-by: Andreas Fuchs <andreas.fuchs@sit.fraunhofer.de>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
...
+### Changed or Fixed
+- FAPI: Fix setting of the system flag of NV objects
+  This will let NV object metadata be created system-wide always instead of
+  locally in the user. Existing metadata will remain in the user directory.
+  It can be moved to the corresponding systemstore manually if needed.
+- FAPI: Set the written flag of NV objects in FAPI PolicyNV commands
+- FAPI: Fix deleting of policy files.
+- FAPI: Fix wrong file loading during object search.
+- Fapi: Fix memory leak
+- Fapi: Fix potential NULL-Dereference
```
Comment (pdxjohnny) [inline CHANGELOG.md:16]: Coverity is showing another potential NULL-Dereference here (just FYI): https://scan8.coverity.com/reports.htm#v22486/p13007/fileInstanceId=37005548&defectInstanceId=11222604&mergedDefectId=312963

image

Comment (AndreasFuchsSIT) [reply CHANGELOG.md:16]: @JuergenReppSIT Could you have a look at this ?

**Comment by JuergenReppSIT** [comment]
I think that's a false positive. Path will only be set after state PROVISION_INIT is activated by the Async method were
pathlist has been set. But I could add a check for path and return "general failuer" if path is equal NULL?

**Comment by JuergenReppSIT** [comment]
@AndreasFuchsSIT I have created PR  #1893 for master with the path check. Should I cherry-pick it for 3.0.x?

**Pull request merged**